engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/Graph.scala

```diff
@@ -24,11 +27,26 @@ sealed class Graph extends Serializable {
     val copy = new Graph
     copy.rootScope     = this.rootScope.deepCopy(scope_mapping)
     copy.links         = this.links
+    copy.sourceLinks   = this.sourceLinks
+    copy.targetLinks   = this.targetLinks
     copy.globalSymbols = this.globalSymbols
     copy.nextIdCounter = this.nextIdCounter
     copy
   }
 
+  def initLinks(links: Set[Graph.Link]): Unit = {
+    links.foreach { link =>
```

Comment:
Shouldn't `sourceLinks` and `targetLinks` be cleared first? Otherwise calling `initLinks` multiple times will accumulate the results. Either:
- allow `initLinks` to be called only once
- clear the content if called multiple times
- rename to `addLinks`